src/leaflike/bookmarks.clj

```diff
@@ -30,26 +27,27 @@
   [{:keys [params] :as request}]
   (let [bookmark (-> (select-keys params [:title :url :tags])
                      format-tags)
-        user    (get-user request)]
+        user    (hutils/get-user request)
+        tags (:tags bookmark)]
     (if (valid-bookmark? bookmark)
-      (let [bookmark (assoc bookmark
-                            :member_id (:id user)
-                            :created_at (utils/get-timestamp))]
-        (bm-db/create bookmark)
+      (let [bookmark (-> bookmark
```

Comment:
@samrat You can thread in `request` instead of `bookmark` and avoid the (probably harmless) shadowing, provided `valid-bookmark` works on the `request` and not 'bookmark` after tags inside has been formatted. It's the input that should be validated anyway, and not after you have formatted it. This also helps you avoid another `let`

Reply:
Fixed this